src/context/apolloClient.js

```diff
@@ -0,0 +1,12 @@
+import { ApolloClient, InMemoryCache, HttpLink } from "@apollo/client";
+
+const httpLink = new HttpLink({
+  uri: "http://your-backend-uri/graphql",
+});
+
+const client = new ApolloClient({
+  link: httpLink,
+  cache: new InMemoryCache(),
+});
+
+export default client;
```

Comment:
This is not needed as its already in the index.js

Reply:
removed